libsolidity/lsp/LanguageServer.cpp

```diff
@@ -212,6 +213,40 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 		if (typeFailureCount)
 			m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
 	}
+
+	Json::Value jsonRemappings = _settings["remappings"];
+
+	if (jsonRemappings)
+	{
+		bool typeFailures = false;
+		if (jsonRemappings.isArray())
+		{
+			vector<frontend::ImportRemapper::Remapping> remappings;
+			for (Json::Value const& jsonPath: jsonRemappings)
+			{
+				if (
+					jsonPath.isObject() &&
+					jsonPath.get("context", "").isString() &&
```

Comment:
New type checking helpers were added and recently merged in [this pull request](https://github.com/ethereum/solidity/pull/13579/files#diff-56f32f3c6823f15db6b0437642c7639a500cf3af3d7a86e16db2073dffca60ceR79), so these should be changed to use `isOfType<T>` or `isOfTypeIfExists<T>`, whichever one is more appropriate.

Reply:
I'd hold off a bit with using these helpers everywhere. I wanted to get them merged not to stall the asm import PR but in the end we may and up removing them (https://github.com/ethereum/solidity/pull/13579#issuecomment-1286803351). Depends on what we end up with after #13579. I mean, it's fine to use them but for now I'd also not force that.